templates/_search-form.html

```diff
@@ -0,0 +1,131 @@
+<form method="GET" class="p-form js-asset-search">
+  {% set params = request.values.to_dict() %}
+  <div class="row">
+
+    <div class="col-4">
+      <label for="tag">Tag</label>
+      <input class="js-tags" type="text" name="tag" placeholder="Tag" id="tag" value="{% if params.tag %}{{ params.tag }}{% endif %}">
+      <button class="p-button--positive" style="margin-top:45px;">Search</button>
+    </div>
+
+    <div class="col-8">
+      <div class="row">
+        {% set asset_types = [
+          "Datasheets", "Guides", "Ebook", "Whitepaper", "Case Study", 
+          "Explainer Video", "Tutorial Video", "Demo", "Infographic", 
+          "Diagram", "Icon", "Illustration", "Image"
+          ] 
+        %}
+        <div class="col-2 js-select">
+          <label for="asset-type">Asset type</label>
+          <select name="asset_type">
+            <option value="">Any type</option>
+            {% for asset_type in asset_types %}
+              {% set asset_value = asset_type|lower|replace(' ', '-') %}
+              <option value="{{ asset_value }}" {% if asset_value in params.asset_type %}selected{% endif %}>
+                {{ asset_type }}
+              </option>
+            {% endfor %}
+          </select>
+        </div>
+
+        {% set languages = [
```

Comment:
could we extract this arrays into a separate file that will be used as a configuration. It would be easier to maintain it in the future. Wdyt?

Reply:
Good idea